src/web/routes/application/check/get.js

```diff
@@ -1,16 +1,97 @@
-const pageContent = {
-  title: 'Check',
-  heading: 'Check'
+const { YES } = require('../are-you-pregnant/constants')
+
+const pageContent = ({ translate, claim }) => ({
+  title: translate('check.title'),
+  heading: translate('check.heading'),
+  sendApplicationHeader: translate('check.sendApplicationHeader'),
+  sendApplicationText: translate('check.sendApplicationText'),
+  buttonText: translate('buttons:acceptAndSend'),
+  checkRowData: buildCheckRowData(translate, claim)
+})
+
+const buildCheckRowData = (translate, claim) => {
+  return [
+    buildNameRow(translate, claim),
+    buildNinoRow(translate, claim),
+    buildDateOfBirthRow(translate, claim),
+    buildAreYouPregnantRow(translate, claim),
+    buildExpectedDeliveryDateRow(translate, claim),
+    buildAddressRow(translate, claim)
+  ].filter(row => row !== undefined)
```

Comment:
Could make use of Ramda `isNil` or javascript `Boolean` here:
```
const buildCheckRowData = (translate, claim) => [
  buildNameRow(translate, claim),
  buildNinoRow(translate, claim),
  buildDateOfBirthRow(translate, claim),
  buildAreYouPregnantRow(translate, claim),
  buildExpectedDeliveryDateRow(translate, claim),
  buildAddressRow(translate, claim)
].filter(!isNil)
```

Reply:
Done